content/_components/webhook/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Webhook component
+layout: article
+section: Utility components
+---
+
+## Description
+
+### Purpose
+An open source component for sending and receiving [WebHooks](https://en.wikipedia.org/wiki/Webhook) on [elastic.io platform](https://www.elastic.io "elastic.io platform").
+
```

Comment:
Please do not use elastic.io here. Use the variable of {{site.data.tenant.name}} instead of the link. So, remove  this part: `[elastic.io platform](https://www.elastic.io "elastic.io platform")`